tests/test_export.py

```diff
@@ -0,0 +1,56 @@
+from .test_create import (make_temp_env, package_is_installed, PYTHON_BINARY,
+                          assert_package_is_installed, run_command, Commands,
+                          make_temp_prefix)
+from os.path import (exists, join, basename)
+from unittest import TestCase
+import tempfile
+import pytest
+
+class ExportIntegrationTests(TestCase):
+
+    @pytest.mark.timeout(900)
+    def test_basic(self):
+        with make_temp_env("python=3") as prefix:
+            assert exists(join(prefix, PYTHON_BINARY))
+            assert_package_is_installed(prefix, 'python-3')
+
+            output, error = run_command(Commands.LIST, prefix, "-e")
+            with tempfile.NamedTemporaryFile(mode="w", suffix="txt", delete=False) as env_txt:
+                env_txt.write(output)
+                env_txt.flush()
+                env_txt.close()
```

Comment:
@kalefranz , If not close the file, on windows , I will get a error like Access denied when read that file during conda create --file

I add rm_rf at the end of with statement.

Thanks